Include through models when preparing

ryanhiebert opened the pull request:
Fix #70, as well as a couple other issues that are worth addressing.

When constructing and preparing models, include the many to many through tables. This adds the `sa` attribute to the through tables, and adds them to the return dictionary from `construct_models`, which is what I was primarily after with this change set.

Deepwalker commented on aldjemy/core.py:
```diff
@@ -73,6 +73,17 @@ def get_meta():
     return Cache.meta
 
 
+def get_tables():
```
This is new function, but I do not see any use of it.

ryanhiebert replied:
I removed this function in #70, but it is documented in the README. Alternative to adding it back, we could remove the reference fr the README and `__init__.__all__`

ryanhiebert replied:
I removed this function and the documentation of it.

Deepwalker commented on aldjemy/orm.py:
```diff
@@ -114,7 +114,7 @@ def _extract_model_attrs(metadata, model, sa_models):
                 remote_side=p_table.c[p_name],
                 )
-            if backref:
+            if backref and not disable_backref:
```
Can you add some test to check if this new code works?

ryanhiebert replied:
I'll see what I can do. I was hitting this bug when crating an alias over a many to many autogenerated through table.

ryanhiebert replied:
I've added a test of how I was using `aliased` and ran across this bug.

Deepwalker commented on aldjemy/orm.py:
```diff
@@ -125,7 +125,7 @@ def _extract_model_attrs(metadata, model, sa_models):
 def prepare_models():
     metadata = get_meta()
-    models = [model for model in get_django_models() if not model._meta.proxy]
+    models = [model for model in get_all_django_models() if not model._meta.proxy]
```
Why do we need this change? Can the difference been tested?

ryanhiebert replied:
I need `construct_models` to return all of the models that were generated, including autogenerated through tables, because I'm doing some manual work with those tables. I should be able to get it tested.

ryanhiebert replied:
I added a bit to the tests to check on the through model to make sure it has an `sa` attribute like all the other models.

ryanhiebert commented:
Thanks, @Deepwalker. Sorry it took me so long to get this cleaned up, I got busy with other priorities at work. Let me know if I've addressed your thoughts sufficiently with the tests I added.

You've previously expressed a desire for freedom from this project. Would you be interested in me and/or my work taking over maintenance of this project?

ryanhiebert commented:
I noticed you did something on #74 , @Deepwalker . Don't forget about me, if you have the time! :smile:

ryanhiebert commented:
Hi @Deepwalker, I hope you're doing well, and that your non-response is due to good fortune rather than calamity. I'm owed nothing, and you've given me quite a lot from this project. Thank you! Would you like me to take over maintenance of this project? Or perhaps it would be better if I were to pull out the stuff I need from this project into a new one? That'll be my default if you're not able to respond, and there will be no hard feelings at all, thank you again! If there's anything I can do to assist you, just let me know!

ryanhiebert commented:
Thanks! This will be very helpful for me to have in a release. Looking forward to it!
